mock/service/service.go

```diff
@@ -185,3 +258,25 @@ func (s *service) getAttachCount(devPathKey string) int64 {
 	}
 	return count
 }
+
+func (s *service) execHook(hookName string) (codes.Code, string) {
+	if s.hooksVm != nil {
+		script := reflect.ValueOf(*s.config.ExecHooks).FieldByName(hookName).String()
+		if len(script) > 0 {
+			result, err := s.hooksVm.Run(script)
+			if err != nil {
+				fmt.Printf("Exec hook %s error -- hook ignored: %v\n", hookName, err)
+				return codes.OK, ""
```

Comment:
This fails on syntax errors and similar stuff. `os.Exit`?

Reply:
The new commid adds os.Exit(1) on error.